src/reducers/index.ts

```diff
@@ -0,0 +1,48 @@
+// tslint:disable-next-line:no-implicit-dependencies
+import { ActionKeys, LoadAction, LoadedAction, TouchAction, TouchedAction } from "@src/actions";
+
+export interface GlobalState {
+    id: string;
```

Comment:
you could consider making these optional as well `id?: string`.  That way you don't have to set them in the initial state.